### ISSUE TITLE
Possibility to get list of panels

### ISSUE THREAD

**Issue by eliogrin** [Invictum/invictum-test#22]
**Example:** I have search result page, where each result is separate panel. I need functionality to get list of panels, for example:
`getAllPanels(SearchResult.class)[1].getSize();`
or 

```
for (SearchResult result in getAllPanels(SearchResult.class)) {
    //do some stuff;
}
```

Is it possible to implement such functionality?

**Comment by Invictum** [comment]
I assume it is possible, but need to check current implementation.

**Comment by Invictum** [comment]
Implemented. Will be available in 0.1.14 version.
